Fix #82 Remove external references

rwakida opened the pull request.

rwakida committed:
Removes kadira connection that made calls to
https://enginex.kadira.io/simplentp/sync

rwakida committed:
Removes account profile options and create user using external services.

rwakida committed:
Replaces browser title with chat locker

rwakida committed:
Removes ability to set avatar using external services.

rwakida committed:
Removes external service packages (e.g. github, linkedin, etc)

rwakida committed:
Removes dependence on externally loaded google fonts.  Roboto font is
loaded locally from files that were downloaded using webfont-dl.

rwakida committed:
Remove unused settings and replaced Rocket.chat references with Chat
Locker

berkich commented:
:+1: Looks good

bbrockman commented on client/views/main.html:
```diff
@@ -9,7 +9,7 @@
+	<meta name="application-name" content="Chat Locker">
 	<meta name="msapplication-TileColor" content="#04436a">
 	<meta name="msapplication-TileImage" content="/images/logo/mstile-144x144.png?v=3">
 	<meta name="msapplication-config" content="/images/logo/browserconfig.xml?v=3">
```
Do we need line 17

```
<link rel="canonical" href="https://demo.rocket.chat/" />
```

bbrockman commented:
Do we need the following files:
 .meteor/cordova-plugins file?
client/lib/constallatoins.js (wasn't this removed from the RC login?
build-android.sh
build-ios.sh
pm2.production.json
/server/methods/getAvatarSuggestion.coffee
/server/startup/migrations/v15.coffee (or at least the method starting on line 62)

Also, do we need all the rocketchat packages? Hubot, external, webrtc, webrtc-ib (maybe others)?

rwakida committed:
Remove webrtc, cordova, fastclick references and packages

rwakida committed:
Merge branch 'master' into feature/chat-locker-82

rwakida committed:
Removed unused mobile builds, background, and avatar request.  Renamed
json config file to example.

rwakida commented:
Please retest.

berkich commented:
Should we make modifications to the names and references in app.json

rwakida committed:
remove all RocketChat code that automatically creates an admin user

rwakida committed:
Removed more external references and unused code
- Removed cordova facebook login
- Removed cordova code
- Removed social links from flex tab area
- Removed login services from client/server
- Cleaned up login form.  Removed register, forgot-password, confirmation
  code.

rwakida committed:
removed demo.rocket.chat reference

rwakida committed:
remove cordova autoupdate

berkich commented:
Successfully tested new installation without network access and browsed the packages/source code for any obvious unneeded code :+1:
